### ISSUE TITLE
Package request: unl0kr

### ISSUE THREAD

**Issue by Anachron** [void-linux/void-packages#51444]
### Package name

unl0kr

### Package homepage

https://gitlab.com/postmarketOS/buffybox/-/tree/master/unl0kr

### Description

Disk unlocker for the initramfs based on LVGL.

### Does the requested package meet the package requirements?

System, Compiled

### Is the requested package released?

Yes

**Comment by Eloitor** [comment]
I'd request the whole BuffyBox

**Comment by Anachron** [comment]
Yep that probably makes sense. The `unl0kr` tool is the bare minimum that I'd love to have.

**Comment by Anachron** [comment]
Upstream seems to be archived, closing.

**Comment by Eloitor** [comment]
It only moved to https://gitlab.postmarketos.org/postmarketOS/buffybox
